build/yamls/flow-aggregator.yml

```diff
@@ -203,6 +213,17 @@ data:
 
       # TLS min version from: VersionTLS10, VersionTLS11, VersionTLS12, VersionTLS13.
       #tlsMinVersion:
+    
+    # clickHouse contains ClickHouse related configuration options.
+    #clickHouse:
+      # DB name containing antrea table.
+      #database: "default"
+      # URL to ClickHouse DB.
+      #dbUrl: "tcp://<click-house-svc>:9000"
+      # Enables DB driver debug log. 
+      #debug: false
+      # Enables LZ4 compresion for batch flow record insert. 
+      #compress: true
```

Comment:
This looks like compress defaults to true while there is no code to override the default value when it's unset so it will actually be true, which may be confusing. Please check how other processes set the default value when it's unset by users.